framework/src/actions/CheckOutputAction.C

```diff
@@ -41,27 +41,24 @@ CheckOutputAction::act()
 void
 CheckOutputAction::checkVariableOutput(const std::string & task)
 {
-  if (_awh.hasActions(task))
+  // Loop through the actions for the given task
```

Comment:
I hope the differ can be smarter to show that only indentation is different here and do not break the block into pieces...

Reply:
Nope!

Reply:
Actually it does, but not by default. You can hide whitespace changes in the "Diff settings".